android/expoview/src/main/java/host/exp/exponent/ReactNativeStaticHelpers.java

```diff
@@ -105,7 +105,7 @@ public static void setExponentNetwork(ExponentNetwork exponentNetwork) {
   }
 
   @DoNotStrip
-  public static OkHttpClient getOkHttpClient(Class callingClass) {
+  public static OkHttpClient getOkHttpUnprefixedClient(Class callingClass) {
```

Comment:
```suggestion
  public static Object getOkHttpClient(Class callingClass) {
```

How about we return an `Object` from this function and depending on `RNObject.versionForClassname` of the `callingClass` we could return either `expolib_v1.okhttp` the way you implemented it in `getOkHttpClient` below or a regular `okhttp` if the calling class is of version 33 or newer?